substrate/pwasm-alloc/src/lib.rs

```diff
@@ -17,19 +16,18 @@ static ALLOCATOR: WasmAllocator = WasmAllocator;
 
 #[cfg(feature = "nightly")]
 mod __impl {
-	extern crate alloc;
 	extern crate pwasm_libc;
 
-	use self::alloc::heap::{GlobalAlloc, Layout, Opaque};
+	use core::alloc::{GlobalAlloc, Layout};
```

Comment:
Do we actually need `feature(alloc)` here?

Reply:
No, it successfully compiled without `alloc` and `allocator_api`.